internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -273,6 +288,44 @@ func (opts *Opts) askDBUserAccessListOptions() error {
 	return nil
 }
 
+func (opts *Opts) askMongoShellQuestion() (bool, error) {
```

Comment:
[nit] this is still really hard to follow, and I think is because you're trying to combine, setting up mongosh paths and if the users wants or not to run mongosh.
I would ask if the user wants to launch mongosh first, it's and easy yes/no question, if they say `no` you can exit early 
If they say yes then you can see if mongosh is set, again, if set, early exit nothing else to do
if not set then you can ask to set up

This goes in line with my previous comment on trying to keep the `if` nesting levels to 1 or two

Reply:
This is the new flow. Hope the code is more clear too.


```
? Insert the cluster name [Press Enter to use the auto-generated name 'QuickstartCluster17'] QuickstartCluster17
? Insert the cloud service provider on which Atlas provisions the hosts AWS
? Select the physical location of your MongoDB cluster US_EAST_1
? Insert the Username for authenticating to MongoDB [Press Enter to use 'andrea_angiolillo'] zx
? Insert the Password for authenticating to MongoDB [Press Enter to use an auto-generated password] [? for help]
? Insert the IP entry to add to the Access List [Press Enter to use your public IP address '212.129.75.137'] 212.129.75.137
? Do you want to access QuickstartCluster17 with MongoDB Shell? Yes
? No MongoDB shell version detected. Do you want to provide the path to your MongoDB shell binary? Yes
? Default MongoDB Shell Path: /usr/local/bin/mongosh
Creating your cluster... [It's safe to 'Ctrl + C']

Now you can connect to your Atlas cluster with: mongo -u zx -p RWt7TlVxVmd7 

mongodb+srv://quickstartcluster17.ajlj3.mongodb-dev.net
Current Mongosh Log ID: 604f9565c64232922543cb60
Connecting to:    mongodb+srv://quickstartcluster17.ajlj3.mongodb-dev.net/
Using MongoDB:      4.4.4
Using Mongosh Beta: 0.8.2

For mongosh info see: https://docs.mongodb.com/mongodb-shell/

Enterprise atlas-11im22-shard-0 [primary]>
```

Please note how now we are always printing this line `Now you can connect to your Atlas cluster with: mongo -u zx -p RWt7TlVxVmd7 `

Reply:
Scenario: user doesn’t want to use mongosh

```
./bin/mongocli atlas quickstart
? Insert the cluster name [Press Enter to use the auto-generated name 'QuickstartCluster19'] QuickstartCluster19
? Insert the cloud service provider on which Atlas provisions the hosts AWS
? Select the physical location of your MongoDB cluster US_EAST_1
? Insert the Username for authenticating to MongoDB [Press Enter to use 'andrea_angiolillo'] dfdf
? Insert the Password for authenticating to MongoDB [Press Enter to use an auto-generated password] [? for help]
? Insert the IP entry to add to the Access List [Press Enter to use your public IP address '212.129.75.137'] 212.129.75.137
? Do you want to access QuickstartCluster19 with MongoDB Shell? No
Creating your cluster... [It's safe to 'Ctrl + C']

Now you can connect to your Atlas cluster with: mongosh -u dfdf -p eWFYdlRFeV5Y mongodb+srv://quickstartcluster19.ajlj3.mongodb-dev.net
```